homework_streams/task_3/task_3.js

```diff
@@ -0,0 +1,18 @@
+const { Readable, Writable } = require("stream");
+
+const readableStream = new Readable({
+  read() {
+    this.push("Anna Tazayan");
+    this.push('21 y/o');
+    this.push("Student at the NPUA and EPAM");
+    this.push("Learning NodeJs");
+    this.push(null);
+  },
+});
+const writableStream = new Writable({
+  write(chunk, enc, cb) {
+    console.log("Write:", chunk.toString());
+    setTimeout(cb, 1000);
+  },
+});
+readableStream.pipe(writableStream)
```

Comment:
The pipe internally handles back pressure, to implement it manually you should use .write method and check the value returned by it, then on `drain` event continue writing again,